src/196AA0.c

```diff
@@ -343,7 +371,15 @@ INCLUDE_ASM(s32, "196AA0", CloseActionCommandInfo);
 
 INCLUDE_ASM(s32, "196AA0", func_80269470);
 
-INCLUDE_ASM(s32, "196AA0", func_802694A4);
+ApiStatus func_802694A4(Evt *arg0, s32 isInitialCall) {
```

Comment:
arg0 should be anmed script, also pointer style on the left (`Evt* script`), also empty line after decls